# HG changeset patch
# Parent 18f41d6ad68350fc7e2bb3bf5903ea3a5db87c58

Xapi doesn't understand cpuid_mask_xsave_eax, and we have to hardwire it to 0
in the installer, causing complaints on all Intel hardware older than SandyBridge.

Reduce the verbosity of the error message until the toolstack is fixed up to
correctly understand these things.

Related to CP-4312

diff -r 18f41d6ad683 xen/arch/x86/cpu/intel.c
--- a/xen/arch/x86/cpu/intel.c
+++ b/xen/arch/x86/cpu/intel.c
@@ -117,7 +117,7 @@ static void __devinit set_cpuidmask(cons
 		return;
 	}
 
-	printk(XENLOG_ERR "Cannot set CPU %sfeature mask on CPU#%d\n",
+	printk(XENLOG_INFO "Cannot set CPU %sfeature mask on CPU#%d\n",
 	       extra, smp_processor_id());
 }
 
